docs/Documentation/Drivers/Classic/OutputFormatting.md

```diff
@@ -20,41 +20,50 @@ VIC allows the user to specify exactly which output files to create and which va
 ```
 # Output File Contents
 OUTFILE	_prefix_
-OUTVAR	_varname_	[_format_	_type_	_multiplier_]
-OUTVAR	_varname_	[_format_	_type_	_multiplier_]
-OUTVAR	_varname_	[_format_	_type_	_multiplier_]
+OUTVAR	_varname_	[_format_	_type_	_multiplier_ [_aggtype_]]
+OUTVAR	_varname_	[_format_	_type_	_multiplier_ [_aggtype_]]
+OUTVAR	_varname_	[_format_	_type_	_multiplier_ [_aggtype_]]
 
 OUTFILE	_prefix_
-OUTVAR	_varname_	[_format_	_type_	_multiplier_]
-OUTVAR	_varname_	[_format_	_type_	_multiplier_]
-OUTVAR	_varname_	[_format_	_type_	_multiplier_]
+OUTVAR	_varname_	[_format_	_type_	_multiplier_ [_aggtype_]]
+OUTVAR	_varname_	[_format_	_type_	_multiplier_ [_aggtype_]]
+OUTVAR	_varname_	[_format_	_type_	_multiplier_ [_aggtype_]]
 ```
 where
 
-_prefix_ = name of the output file, NOT including latitude and longitude
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; _prefix_ = name of the output file, NOT including latitude and longitude
 
-_varname_ = name of the variable (this must be one of the output variable names listed in `vic_driver_shared.h`.)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; _varname_ = name of the variable (this must be one of the output variable names listed in `vic_driver_shared.h`.)
 
-_format_, _type_, and _multiplier_ are optional.  For a given variable,
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; _format_, _type_, and _multiplier_ are optional.  For a given variable,
```

Comment:
I wanted these lines to be indented, so that they look clearer on the website. And this is the only way I found to control indentation in markdown... can remove the indentation if it's not desired